tests/rules/severity_critical.yaml

```diff
@@ -0,0 +1,7 @@
+rules:
+- id: critical
+  languages:
+  - python
+  match: stuff($ARG);
+  message: rule_template_message
+  severity: CRITICAL  
```

Comment:
Should we add a test for each level so that we get a failure if the mappings fail?